tests/squash.bats

```diff
@@ -61,53 +66,59 @@ function check_lengths() {
 		echo COPY randomfile /layer${stage} >> ${TESTDIR}/stage${stage}/Dockerfile
 		image=stage${stage}
 		from=${image}
-		buildah build-using-dockerfile --signature-policy ${TESTSDIR}/policy.json -t ${image} ${TESTDIR}/stage${stage}
+		run_buildah build-using-dockerfile --signature-policy ${TESTSDIR}/policy.json -t ${image} ${TESTDIR}/stage${stage}
                 check_lengths $image $stage
 	done
 
 	mkdir -p ${TESTDIR}/squashed
 	echo FROM ${from} > ${TESTDIR}/squashed/Dockerfile
 	cp ${TESTDIR}/randomfile ${TESTDIR}/squashed/
 	echo COPY randomfile /layer-squashed >> ${TESTDIR}/stage${stage}/Dockerfile
-	buildah build-using-dockerfile --signature-policy ${TESTSDIR}/policy.json --squash -t squashed ${TESTDIR}/squashed
+	run_buildah build-using-dockerfile --signature-policy ${TESTSDIR}/policy.json --squash -t squashed ${TESTDIR}/squashed
 
         check_lengths squashed 1
 
-	cid=$(buildah from squashed)
-	mountpoint=$(buildah mount $cid)
+	run_buildah from --quiet squashed
+	cid=$output
+	run_buildah mount $cid
+	mountpoint=$output
 	for stage in $(seq 10) ; do
 		cmp $mountpoint/layer${stage} ${TESTDIR}/randomfile
 	done
 
-	buildah build-using-dockerfile --signature-policy ${TESTSDIR}/policy.json --squash --layers -t squashed ${TESTDIR}/squashed
-	run_buildah --log-level=error inspect -t image -f '{{len .Docker.RootFS.DiffIDs}}' squashed
-	[ "$output" -eq 1 ]
+	run_buildah build-using-dockerfile --signature-policy ${TESTSDIR}/policy.json --squash --layers -t squashed ${TESTDIR}/squashed
```

Comment:
Hey, so, this here has `--layers` and I kind of assumed that the subsequent tests would also, but they don't. Is that intentional?

Reply:
Makes sense to me.

Reply:
As in, "makes sense that the rest should also specify `--layers`", or "makes sense the way it is"?

Reply:
All  should have --layers in this test.  Or none should have layers.  But I think we want at least some tests specifying --layers.

Reply:
We want at least a mix of `--squash` and `--squash --layers` within the tests, we don't want one or the other for all of the tests.  For more info on the expected behavior, you can check out the description in #1900

Reply:
I just realized I added my github comment in the wrong place. What I was struggling with is trying to figure out the difference between lines 93-107 and 109-123. Ignoring my test-name changes, the only diff between those two blocks is that line 109 has `--layers`. The rest of the `bud` steps don't. Is that intentional?

Reply:
@edsantiago over time there's been a lot of pruning done here and there.  I try to keep at least one of each option scattered about, most likely it was intentional at one point, but I don't know the why it was added (or left behind) at this point.